pkg/preparer/orchestrate.go

```diff
@@ -175,6 +175,10 @@ func (p *Preparer) installAndLaunchPod(newManifest *pods.PodManifest, pod Pod, l
 	}
 
 	if newOrDifferent || problemReadingCurrentManifest {
+		err := p.iStore.RegisterPodService(*newManifest)
+		if err != nil {
+			logger.WithField("err", err).Errorln("Service registration failed")
```

Comment:
What are the error cases when the intent store registration fails? Just want to hear the thinking on not bailing out / continuing onto `Launch`